docs/product_search/index.md

```diff
@@ -3,4 +3,82 @@ layout: default
 title: Product Search
 has_children: true
 nav_order: 8
+has_toc: false
 ---
+
+# Product Search
+Product search is based on query parameters that determine which products are returned and in which order. In addition to the search query itself, there are parameters that allow you to customize the search further, for example you could:
+
+- Choose whether unavailable products are returned
+- Return only products with active promotions
+- Return products for a specific date range or year
+- Sort the results by departure date
+
+The product search is executed using the [product_search tag]({% link docs/reference/tags/product_search/index.md %}). The tag returns an `items` array of [products]({% link docs/reference/objects/product/index.md %}), a `paginate` [pagination]({% link docs/reference/objects/pagination.md %}) object and  a `search` search object. 
```

Comment:
What is the search object?

Reply:
I found this one in the other docs and copied over, have noted down to add a page in references for it